src/qt/test/wallettests.cpp

```diff
@@ -184,6 +191,69 @@ void TestSendCoins()
     BumpFee(transactionView, txid2, false /* expect disabled */, {} /* expected error */, false /* cancel */);
     BumpFee(transactionView, txid2, true /* expect disabled */, "already bumped" /* expected error */, false /* cancel */);
 
+    // Check current balance on OverviewPage
+    OverviewPage overviewPage(platformStyle.get());
+    overviewPage.setWalletModel(&walletModel);
+    QLabel* balanceLabel = overviewPage.findChild<QLabel*>("labelBalance");
+    QString balanceText = balanceLabel->text();
+    int unit = walletModel.getOptionsModel()->getDisplayUnit();
+    CAmount balance = walletModel.getBalance();
+    QString balanceComparison = BitcoinUnits::formatWithUnit(unit, balance, false, BitcoinUnits::separatorAlways);
+    QCOMPARE(balanceText, balanceComparison);
+
+    // Check Request Payment button
+    ReceiveCoinsDialog receiveCoinsDialog(platformStyle.get());
+    receiveCoinsDialog.setModel(&walletModel);
+    RecentRequestsTableModel* requestTableModel = walletModel.getRecentRequestsTableModel();
+
+    // Label input
+    QLineEdit* labelInput = receiveCoinsDialog.findChild<QLineEdit*>("reqLabel");
+    labelInput->setText("TEST_LABEL_1");
+
+    // Amount input
+    BitcoinAmountField* amountInput = receiveCoinsDialog.findChild<BitcoinAmountField*>("reqAmount");
+    CAmount testAmount = 1.0;
```

Comment:
`CAmount`s are not floats; just use `1`. Since you're not using it elsewhere you may want to just do `setValue(1)` below.